BUILD.bazel

```diff
@@ -9,6 +9,15 @@ cc_library(
     visibility = ["//visibility:public"],
 )
 
+cc_library(
+    name = "array_absl",
+    hdrs = ["array_absl.h"],
```

Comment:
Any objections to the name for the target and the header?

I think the target is fine but the header could be more fine grained if we want (array_absl_stringify.h?)